src/tests/multihost/alltests/test_krb_access_provider.py

```diff
@@ -0,0 +1,177 @@
+"""Automation for krb access provider
+"""
+from __future__ import print_function
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.exceptions import SSSDException
+from constants import ds_instance_name
+
+
+@pytest.mark.krbaccessprovider
+@pytest.mark.usefixtures('create_custom_posix_usersgroups')
+class TestKrbAccessProvider(object):
+    """
+    This is test case class for krb_access_provider suite
+    """
+    def test_0001_k5login_empty(multihost):
+        """
+        """
+        paths = ["/var/log/secure", 
+                 "/var/log/sssd/sssd_%s.log" % ds_instance_name]
+        for path in paths:
+            erase_file(path, multihost)
+
+        sssdTools(multihost.client[0]).clear_sssd_cache()
+
+        cmds = ["mkdir -p /home/testuser3", "chown testuser3: /home/testuser3",
+                "chmod 700 /home/testuser3"]
+        for command in cmds:
+            multihost.client[0].run_command(command)
+
+        path = "/home/testuser3/.k5login"
+        erase_file(path, multihost)
+
+        chown_cmd = "chown testuser3 /home/testuser3/.k5login"
+        restore_selinux = "restorecon -v /home/testuser3/.k5login"
+        multihost.client[0].run_command(chown_cmd)
+        multihost.client[0].run_command(restore_selinux)
+
+        lookup_u = "auth_failure testuser3 Secret123"
+        cmd = multihost.client[0].run_command(lookup_u)
+        assert cmd.returncode == 0
+
+        time.sleep(30)
+
+        strings = ["pam_sss(sshd:auth): authentication success",
+                   "pam_sss(sshd:account): Access denied for user testuser3",
+                   "Access denied for user testuser3"]
+        logfiles = ['/var/log/secure', "/var/log/secure",
+                    '/var/log/sssd/sssd_%s.log' % ds_instance_name]
+
+        for index, string in enumerate(strings):
```

Comment:
You are iterating over strings but use the string index to select item from logfiles, that does not make any sense and it will crash on index error if they do not have the same number of items.

Reply:
Please just download the two logs and add the appropriate asserts:
No loops needed:

```
secure_log_str = multihost.client[0].get_file_contents(
            "/var/log/secure").decode('utf-8')
assert "pam_sss(sshd:auth): authentication success" in secure_log_str, "authentication success not found in /var/log/secure"
...
```